tests/queries/0_stateless/02477_s3_request_throttler.sql

```diff
@@ -0,0 +1,15 @@
+-- Tags: no-fasttest
+-- Tag no-fasttest: needs s3
+
+-- Limit S3 PUT request per second rate
+SET s3_max_put_rps = 2;
+SET s3_max_put_burst = 1;
+
+CREATE TEMPORARY TABLE times (t DateTime);
+
+-- INSERT query requires 3 PUT requests (CreateMultipartUpload, UploadPart, CompleteMultipartUpload) and 1/rps = 0.5 second in between, the first query is not throttled due to burst
```

Comment:
May be add assertion for this using profile event S3PutObject?

Reply:
done